libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/ActionButtonDescription.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.navigation.dropin
+
+import android.view.View
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Custom Action Button description.
+ *
+ * @param view a view that will be added to the [NavigationView]
+ * @param position determines if the custom button should be placed before or after existing controls.
+ *                Buttons will be placed in the following order:
+ *                ```(0, 1, ..., (default action buttons), -1, -2, ...)```
+ *
+ *                Defaults to -1.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class ActionButtonDescription(
+    val view: View,
+    val position: Int = -1
```

Comment:
Since we are preparing a list of `ActionButtonDescription` later in the process, why not make use of the insertion order in the list itself? We could then change this data structure to 

```
class ActionButtonDescription(
    val view: View,
    val position: ActionButtonPosition = ActionButtonPosition.Bottom
)
```
where `ActionButtonPosition` can be

```
sealed class ActionButtonPosition {
    object Top: ActionButtonPosition()
    object Bottom: ActionButtonPosition()
}
```
This way developers don't need to define numerical positions. All they specify is `Top` or `Bottom` relative to default buttons and we add these custom buttons to `Top` or `Bottom` in the order they were added to the list.

Reply:
@abhishek1508 Sorry. Just noticed your comment after already updating it.
Please let me know if my solution is satisfactory.